recipes/cyrsoxs/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "cyrsoxs" %}
+{% set version = "1.1.5.0" %}
+{% set sha = "7ce6e64d5e794751476e6e1b6dbb1cd940b2dd26796b8d416fd82b6c20302ff9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/cyrsoxs/archive/refs/tags/V{{ version }}.tar.gz
+  sha256: {{ sha }}
```

```suggestion
{% set version = "1.1.5.0" %}

package:
  name: cyrsoxs
  version: {{ version }}

source:
  url: https://github.com/usnistgov/cyrsoxs/archive/refs/tags/V{{ version }}.tar.gz
  sha256: 7ce6e64d5e794751476e6e1b6dbb1cd940b2dd26796b8d416fd82b6c20302ff9
```

Final cleanup. Many of us think that the `name` and `sha` variables is overdoing it. They aren't much more usefull than simply using the name of your package everywhere.

Anyway, this is something you can fix yourself in your feedstock. I'll come back around in an hour or so an merge whatever the status.

---

Sounds good, I agree, it's a little silly...the name isn't exactly going to change and sha is only in one place.  I'll commit this.  Really appreciate the review!